tests/integration/test_authd/data/authd_key_hash.yaml

```diff
@@ -0,0 +1,72 @@
+# Tests with agent's key hash. Key generated
+-
+  name: 'Testing general parsing options with key hash 1'
+  description: 'Registering new agent with any hash'
+  test_case:
+  -
+    input: "OSSEC A:'NewUserWithKeyHash1' K:'123ABCD'"
+    output: "OSSEC K:'001 NewUserWithKeyHash1"
```

Comment:
In fact, this test only compares a sub-string of the output, because the random password of the manager can'be anticipated. It happens here `assert response[:len(expected)]`.

The complete output would be something like `"OSSEC K:'001 NewUserWithKeyHash1 RandomPASS'"`, so the comparison ends with the user name.